web/src/components/molecules/Content/Table/index.tsx

```diff
@@ -37,13 +37,7 @@ import {
 import { ContentTableField, Item } from "@reearth-cms/components/molecules/Content/types";
 import { Request } from "@reearth-cms/components/molecules/Request/types";
 import { CurrentViewType } from "@reearth-cms/components/organisms/Project/Content/ContentList/hooks";
-import {
-  SortDirection,
-  FieldSelector,
-  FieldType,
-  ItemSortInput,
-  ConditionInput,
-} from "@reearth-cms/gql/graphql-client-api";
+import { SortDirection, FieldSelector, FieldType } from "@reearth-cms/gql/graphql-client-api";
```

Comment:
GraphQL types shouldn't be used in molecules.
ViewModel types with a converter function to convert from GraphQL type to the ViewModel type should be created.
Please check the reearth frontend development guide for reference.